### ISSUE TITLE
ayatana-ido-0.9.2 fails to build with tests enabled

### ISSUE THREAD

**Issue by pacho2** [AyatanaIndicators/ayatana-ido#62]
I get lots of undefined references:
```
[35/35] : && /usr/bin/x86_64-pc-linux-gnu-g++ -O2 -pipe -march=native -fno-stack-protector -Wl,-O1 -Wl,--as-needed     -no-pie tests/CMakeFiles/gtest-menuitems.dir/gtest-menuitems.cpp.o -o tests/gtest-menuitems  -Wl,-rpath,/var/tmp/portage/dev-libs/ayatana-ido-0.9.2/work/ayatana-ido-0.9.2_build/src  src/libayatana-ido3-0.4.so.0.0.0  -lgtk-3  -lgdk-3  -lpangocairo-1.0  -lpango-1.0  -lharfbuzz  -latk-1.0  -lcairo-gobject  -lcairo  -lgdk_pixbuf-2.0  -lgio-2.0  -lgobject-2.0  -lglib-2.0  -lm && :
FAILED: tests/gtest-menuitems 
: && /usr/bin/x86_64-pc-linux-gnu-g++ -O2 -pipe -march=native -fno-stack-protector -Wl,-O1 -Wl,--as-needed     -no-pie tests/CMakeFiles/gtest-menuitems.dir/gtest-menuitems.cpp.o -o tests/gtest-menuitems  -Wl,-rpath,/var/tmp/portage/dev-libs/ayatana-ido-0.9.2/work/ayatana-ido-0.9.2_build/src  src/libayatana-ido3-0.4.so.0.0.0  -lgtk-3  -lgdk-3  -lpangocairo-1.0  -lpango-1.0  -lharfbuzz  -latk-1.0  -lcairo-gobject  -lcairo  -lgdk_pixbuf-2.0  -lgio-2.0  -lgobject-2.0  -lglib-2.0  -lm && :
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: /usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../lib64/crt1.o: in function `_start':
(.text+0x17): undefined reference to `main'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: tests/CMakeFiles/gtest-menuitems.dir/gtest-menuitems.cpp.o: in function `TestMenuitems_BuildCalendar_Test::TestBody()':
gtest-menuitems.cpp:(.text+0x119): undefined reference to `testing::Message::Message()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x140): undefined reference to `testing::internal::GetBoolAssertionFailureMessage[abi:cxx11](testing::AssertionResult const&, char const*, char const*, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x161): undefined reference to `testing::internal::AssertHelper::AssertHelper(testing::TestPartResult::Type, char const*, int, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x16c): undefined reference to `testing::internal::AssertHelper::operator=(testing::Message const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x174): undefined reference to `testing::internal::AssertHelper::~AssertHelper()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x1ee): undefined reference to `testing::Message::Message()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x215): undefined reference to `testing::internal::GetBoolAssertionFailureMessage[abi:cxx11](testing::AssertionResult const&, char const*, char const*, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x236): undefined reference to `testing::internal::AssertHelper::AssertHelper(testing::TestPartResult::Type, char const*, int, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x241): undefined reference to `testing::internal::AssertHelper::operator=(testing::Message const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x249): undefined reference to `testing::internal::AssertHelper::~AssertHelper()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x2d9): undefined reference to `testing::Message::Message()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x300): undefined reference to `testing::internal::GetBoolAssertionFailureMessage[abi:cxx11](testing::AssertionResult const&, char const*, char const*, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x321): undefined reference to `testing::internal::AssertHelper::AssertHelper(testing::TestPartResult::Type, char const*, int, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x32c): undefined reference to `testing::internal::AssertHelper::operator=(testing::Message const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x334): undefined reference to `testing::internal::AssertHelper::~AssertHelper()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x3b6): undefined reference to `testing::Message::Message()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x3dd): undefined reference to `testing::internal::GetBoolAssertionFailureMessage[abi:cxx11](testing::AssertionResult const&, char const*, char const*, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x3fe): undefined reference to `testing::internal::AssertHelper::AssertHelper(testing::TestPartResult::Type, char const*, int, char const*)'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x409): undefined reference to `testing::internal::AssertHelper::operator=(testing::Message const&) const'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: gtest-menuitems.cpp:(.text+0x411): undefined reference to `testing::internal::AssertHelper::~AssertHelper()'
/usr/lib/gcc/x86_64-pc-linux-gnu/11.3.0/../../../../x86_64-pc-linux-gnu/bin/ld: tests/CMakeFiles/gtest-menuitems.dir/gtest-menuitems.cpp.o: in function `TestMenuitems_BuildEntry_Test::TestBody()':
gtest-menuitems.cpp:(.text+0x659): undefined reference to `testing::Message::Message()'
```

Thanks 

[build.txt](https://github.com/AyatanaIndicators/ayatana-ido/files/10238093/build.txt)

**Comment by JamiKettunen** [comment]
Should be fixed with #63 which is now tagged as [`0.9.3`](https://github.com/AyatanaIndicators/ayatana-ido/releases/tag/0.9.3)

**Comment by pacho2** [comment]
It works fine

Thanks!
